cmd/nodeadm/install/install.go

```diff
@@ -81,9 +82,9 @@ func (c *command) Run(log *zap.Logger, opts *cli.GlobalOptions) error {
 
 	// Default containerd source to distro
 	if c.containerdSource == "" {
-		c.containerdSource = string(containerd.ContainerdSourceDistro)
+		c.containerdSource = string(tracker.ContainerdSourceDistro)
 	}
```

Comment:
imo the right way to do this is to define the default in the `cmd` variable in line 32. That way it shows up in the command help

I know this is not your code though